src/features/contactmoment/ContactmomentNotitie.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
```

Comment:
Kunnen we zonder deze div?

Reply:
nee. dan moet hij naar de parent. als je hem op meerdere plekken gebruikt moet je dan steeds een div toevoegen

Reply:
besproken en aangepast